common/templates/grant_data_sharing_permissions.html

```diff
@@ -3,13 +3,20 @@
 <%!
 from django.core.urlresolvers import reverse
 from django.utils.translation import ugettext as _
+from openedx.core.djangolib.js_utils import dump_js_escaped_json
 %>
 
-<!--<%namespace name='static' file='/static_content.html'/>-->
-
 <%inherit file="/main.html" />
 <%namespace name='static' file='/static_content.html'/>
 
+<script  type="text/javascript">
```

Comment:
Is this the right place to add this function?  Should it instead be added to the JS application layer?